examples/dt/bgp/control-plane/service-values.yaml

```diff
@@ -19,6 +19,8 @@ data:
       swift_store_endpoint_type = internalURL
       swift_store_user = service:glance
       swift_store_key = {{ .ServicePassword }}
+      [database]
```

Comment:
don't we need debug = True here before line 12?

Reply:
yep thank you!